src/EnrollWindowsHelloController.js

```diff
@@ -0,0 +1,150 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define([
+  'okta',
+  'util/FormController',
+  'util/FormType',
+  'vendor/webauthn',
+  'vendor/lib/q',
+  'views/enroll-factors/Footer',
+  'views/mfa-verify/WindowsHelloErrorMessageView'
+],
+function (Okta, FormController, FormType, webauthn, Q, Footer, WindowsHelloErrorMessageView) {
+
+  var _ = Okta._;
+
+  return FormController.extend({
+    className: 'enroll-windows-hello',
+    Model: {
+      local: {
+        __isEnrolled: 'boolean'
```

Comment:
usually our local variables have ```__``` before and after, so this should be ```__isEnrolled__```, or maybe just ```__enrolled__```